glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -78,6 +81,10 @@ open class GleanInternalAPI internal constructor () {
         this.gleanDataDir = File(applicationContext.applicationInfo.dataDir, GLEAN_DATA_DIR)
         handle = LibGleanFFI.INSTANCE.glean_initialize(this.gleanDataDir.path, applicationContext.packageName)
```

Comment:
Sorry for not catching this before, but can we change `glean_initialize`  to have `uploadEnabled` passed in here? The outer interface will change later on, and it will make our intent more explicit internally. Less errors, hopefully.